### PR TITLE
Revert "Updated to pipenv 2020.8.13 and locked"

### PR DESCRIPTION
**Pull request by saisankargochhayat**
Reverts thoth-station/kebechet#510

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 click
 daiquiri
-pipenv==2020.8.13
+pipenv
```
Comment (saisankargochhayat) [inline requirements.txt:3]: should this be locked at 2018?

Comment (fridex) [reply requirements.txt:3]: Yes, I think so. You can do it in a separate PR.

Comment (saisankargochhayat) [reply requirements.txt]: Sorry I added it here.

Comment (fridex) [reply requirements.txt]: Even better :+1:

**Review by fridex — APPROVED** [review]

**Pull request merged**